### ISSUE TITLE
darkflow results differ from darknet

### ISSUE THREAD

**Issue by glenn-jocher** [thtrieu/darktf#718]
I see different results when running darkflow and darknet on the same image, using the same default *.weight and *.cfg files. 

Darknet (black) appears more accurate than darkflow (white). The dog is notably missed in both images by darkflow, but successfully detected by darknet. I've made no changes to the default *.cfg file, threshold is 0.6, input dimension is 416. 

Darknet confidences are generally higher with better bounding boxes as well.

[/darknet/data/](https://github.com/pjreddie/darknet/tree/master/data)dog.jpg with [yolov2-tiny.weights](https://pjreddie.com/media/files/yolov2-tiny.weights) and [/darknet/cfg/](https://github.com/pjreddie/darknet/tree/master/cfg)yolov2-tiny.cfg
![dog yolov2-tiny](https://user-images.githubusercontent.com/26833433/38753466-e165006a-3f34-11e8-87e6-2c892916bbd0.jpg)

[/darknet/data/](https://github.com/pjreddie/darknet/tree/master/data)person.jpg with [yolov2-tiny.weights](https://pjreddie.com/media/files/yolov2-tiny.weights) and [/darknet/cfg/](https://github.com/pjreddie/darknet/tree/master/cfg)yolov2-tiny.cfg
![person yolov2-tiny](https://user-images.githubusercontent.com/26833433/38753452-dae23302-3f34-11e8-8d00-1ebb0697b80a.jpg)

**Comment by lfb94** [comment]
Hello,

Did you figure this out?

Thanks

**Comment by wns349** [comment]
Same problem here. 
I changed `weights_walker`'s starting offset from 16 to 20 to get it running with `yolov2-tiny.weights`.
I am assuming that's not the problem here though.

**Comment by glenn-jocher** [comment]
@wns349 yes the starting offset must be changed from 16 to 20 before running `yolov2-tiny`. I don't think this is the issue though, as I also see difference in `yolov2` with the default offset of 16. I think there might be structural inconsistencies in the darkflow implementation that cause it to differ.

**Comment by Malouke** [comment]
i am real disappointed about darkflow i spend 3 weeks on labelling and training data but without result even with million images of cells and annotations really not good
